dispatcher/dispatcher.py

```diff
@@ -86,9 +32,7 @@ def main():
     args = parser.parse_args()
     config = configparser.ConfigParser()
     config.read(args.config)
-    logger = logging.getLogger('main')
-    logger.addHandler(LogHandler())
-    logger.setLevel(getattr(logging, args.log))
+    logger = get_daq_logger('dispatcher', level=getattr(logging, args.log))
```

Comment:
want to have name = 'main', process_name='dispatcher'?

Reply:
We don't actually use `main` anywhere in the logging output (only the time, level, and message), so I don't think it makes a lot of difference. If we wanted this logger to get used in a much more structured process then we would be using this field, but in the current use cases I think it's largely unnecessary.